sjb/config/common/test_cases/origin_release_install_azure.yml

```diff
@@ -0,0 +1,190 @@
+---
+parent: 'common/test_cases/origin_release_with_ecosystem.yml'
+extensions:
+  sync:
+    - "openshift,release=master"
+  parameters:
+    - name: "CLUSTER_PROFILE"
+      description: "The subdirectory of openshift/release cluster/test-deploy to use for configuration of the cluster."
+      default_value: "gcp"
```

Comment:
is this shared? If this is the azure job then maybe this should be azure as its default. If this is shared then gcp might be ok